### ISSUE TITLE
LoRa not working, no Chirpstack

### ISSUE THREAD

**Issue by cdupont** [Waziup/WaziGate#128]
No Lora functions are working, Chirpstack is not responding on http://wazigate.local:8080/

Logs postgreSQL:
```
$ docker logs postgresql
2020-12-07T16:44:21.549773771Z FATAL:  no pg_hba.conf entry for host "172.18.0.2", user "chirpstack_as", database "chirpstack_as", SSL off
2020-12-07T16:44:23.549502783Z FATAL:  no pg_hba.conf entry for host "172.18.0.3", user "chirpstack_ns", database "chirpstack_ns", SSL off
2020-12-07T16:44:23.554703267Z FATAL:  no pg_hba.conf entry for host "172.18.0.2", user "chirpstack_as", database "chirpstack_as", SSL off
```

Adding this line to pg_hba.conf seems to solve the problem:
```
$ docker exec -it postgresql sh
# echo "host    all             all             0.0.0.0/0            trust" >> ./var/lib/postgresql/data/pg_hba.conf
```
Not sure it's Kosher, though.

**Comment by cdupont** [comment]
A better option is to remove the postgreSQL volume:
```
cd ~/waziup-gateway/apps/waziup/wazigate-lora
docker-compose down
docker volume rm wazigate-lora_postgresqldata
sudo reboot
```
After rebooting, the volume should be re-created and the DB will be initialized correctly by wazigate-lora.

**Comment by j-forster** [comment]
```
cd ~/waziup-gateway/apps/waziup/wazigate-lora
docker-compose down
docker volume rm wazigate-lora_postgresqldata
docker-compose up -d
```
No need to restart here.

**Comment by cdupont** [comment]
Even better:
```
sudo docker system prune --volumes -f
```

**Comment by cdupont** [comment]
This is fixed in ISO 2.1.3
Closing
